css-brochure-form/styles.css

```diff
@@ -0,0 +1,61 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+h1 {
+  text-align: center;
+  color: #FF7468;
+  text-transform: uppercase;
+  font-family: 'Brandon Grotesque';
+  letter-spacing: .4rem;
+}
+
+.info {
+  font-family: 'Brandon Grotesque';
+  font-style: normal;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.box {
+  width: 600px;
+  height: 40px;
+  border-color: lightgrey;
+  border-radius: 3px;
+  margin-bottom: 20px;
+  margin-top: 5px;
+  border-width: 1px;
+  font-family: 'Brandon Grotesque';
+  font-style: normal;
+  font-size: 16px;
+}
+
+.hire {
+  font-family: 'Brandon Grotesque';
+  font-style: normal;
+  font-size: 16px;
+}
+
+.sub {
+  margin-left: 500px;
```

Comment:
while margin can be used like this, it should not be. Margin is best avoided for layout position. I would recommend using the text-align property on the buttons parent with the appropriate value. here are the docs: https://developer.mozilla.org/en-US/docs/Web/CSS/text-align